lib/ansible/modules/notification/irc.py

```diff
@@ -185,14 +191,15 @@ def send_msg(msg, server='localhost', port='6667', channel=None, nick_to=None, k
     if use_ssl:
         irc = ssl.wrap_socket(irc)
     irc.connect((server, int(port)))
+
     if passwd:
-        irc.send('PASS %s\r\n' % passwd)
-    irc.send('NICK %s\r\n' % nick)
-    irc.send('USER %s %s %s :ansible IRC\r\n' % (nick, nick, nick))
+        irc.send(safe('PASS %s\r\n' % passwd))
+    irc.send(safe('NICK %s\r\n' % nick))
+    irc.send(safe('USER %s %s %s :ansible IRC\r\n' % (nick, nick, nick)))
     motd = ''
     start = time.time()
     while 1:
-        motd += irc.recv(1024)
+        motd += to_text(irc.recv(1024))
```

Comment:
This looks okay but if you do this then you should probably make the regex string explicitly text type instead of native type.